apps/condo/pages/reports/external/[id]/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import { Row, Col, Typography } from 'antd'
+import { PageContent, PageHeader, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
+import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
+import { ExternalReport } from '@condo/domains/analytics/utils/clientSchema'
+import { Loader } from '@condo/domains/common/components/Loader'
+
+const LAYOUT_STYLE: React.CSSProperties = { height: '100%' }
+const IFRAME_STYLE: React.CSSProperties = { border: 'none', width: '100%', height: '100%' }
+
+const ExternalReportDetailPage = () => {
+    const { asPath } = useRouter()
+
+    const {
+        obj: externalReport, loading,
+    } = ExternalReport.useObject({
+        where: {
+            id: asPath.split('/').pop(),
+        },
+    }, { fetchPolicy: 'network-only' })
```

Comment:
why network only?

Reply:
Because `iframeUrl` virtual field has expiration time. And on each request this field need's to be up to date and not had been cached at all